openapi-spec/swagger.yaml

```diff
@@ -87,7 +92,104 @@ tags:
     externalDocs:
       description: Find out more
       url: 'https://sodafoundation.io/'
+# schemes:
+# - http
 paths:
+  '/v1/{project_id}/aksks':
```

Comment:
could not see any resource with project? is project_id same as tenant_id? or do we have another API for GET project? all other APIS I see tenant_id

Reply:
its same as tenant Id

Reply:
then better to follow tenent_id, as all other APIS has tenant_id

Reply:
as its a Keystone API , it uses "project_id",  Using both "project_id and tenant_id will be confusing, 
So , ticking with "project_id" to be consistent with Keystone, In the documentation / help I can mention that project_id is same as tenant_id

Reply:
ok

Reply:
Well.. at best its an integration point of SODA with Keystone .. so its a grey area. Neither fully in SODA nor Keystone. So , IMO for the sake of consistency , its best the Originator (Keystones') terminologies be used.